calibrations/tpc/gemcurrents/daqdbreader_tpc_GEM_charge_status_2.py

```
@@ -0,0 +1,160 @@
+    # Prometheus query
+    # ---------------------------
+    for variable_name in variable_names:
+
+        params = {
+            'query': variable_name,
+            'start': begin_run,
+            'end': end_run,
+            'step': '1m'
+        }
+
+        resp = requests.get(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)